finances/templates/finances/js/sharedevent_create_form_logic.html

```diff
@@ -0,0 +1,13 @@
+{% block content %}
+<script type="text/javascript">
+$(function(){
+  $("#id_allow_self_registeration").change(function() {
+    if (this.checked) {
+        $("#id_date_end_registration").prop('disabled', false);
+    } else {
+	    $("#id_date_end_registration").prop('disabled', true);
+	}
+  });
+});
+</script>
+{% endblock %}
```

Comment:
This is full js, right ?  It should then go to the static folder :)

Reply:
Well, you are right!